LiteCore/Database/SequenceTracker.hh

```diff
@@ -20,7 +20,7 @@
 
 namespace litecore {
     class CollectionChangeNotifier;
-    class DatabaseImpl;
+    class CollectionImpl;
```

Comment:
This change must be unnecessary, since the file compiled without it. And in fact `class DatabaseImpl` must be unnecessary too if you could change it without breaking anything, so this line can be deleted.